### PR TITLE
Fix/ledger integration

### PR DESCRIPTION
**Pull request by micovi**
This PR replaces Ledger Transport U2F and Ledger Transport WebAuthn in favor of WebUSB to fix Ledger Devices connections on newest Chromium browsers.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
   "dependencies": {
     "@ledgerhq/hw-transport-u2f": "^5.13.1",
     "@ledgerhq/hw-transport-webauthn": "^5.15.0",
```
Comment (noel-yoo) [inline package.json:12]: Remove the unused dependencies:
```
"@ledgerhq/hw-transport-u2f": "^5.13.1",
"@ledgerhq/hw-transport-webauthn": "^5.15.0",
```

**Review by noel-yoo — CHANGES_REQUESTED** [review]

**Comment by bb111189** [comment]
> This PR replaces Ledger Transport U2F and Ledger Transport WebAuthn in favor of WebUSB to fix Ledger Devices connections on newest Chromium browsers.

Can WebUSB work on FF? I remember we need to use U2F on FF and WebUSB on chromium based browsers

**Comment by noel-yoo** [comment]
check this https://caniuse.com/?search=WebUSB @bb111189

**Comment by bb111189** [comment]
> check this https://caniuse.com/?search=WebUSB @bb111189

Yes. I think for FF user agents, we need to stick to U2F. The rest, we can use WebUSB

File: package.json (hunk 1 above)
Comment (micovi) [reply package.json]: Removed the webauthn dep

**Comment by micovi** [comment]
Added a check if there is WebUSB supported then fallback to U2F for FF browsers.
Removed webauthn dep.
Also added the check on Import account.

**Review by noel-yoo — APPROVED** [review]

**Pull request merged**